script.js

```diff
@@ -85,12 +93,20 @@ function getBeerRec() {
 }
 
 function displayBeers(responseJson, targetedDiv) {
-    for (let i = 0; i < 3; i++) {
-        $(targetedDiv).append(`<div class="beers"><h3>Beer Recomendations based of your recipe....</h3>
-          <h4>${responseJson[i].name}</h4>
-          <img src="${responseJson[i].image_url}" alt="" width="20%">
-          <p>${responseJson[i].description}</p></div>`)
+    // we can't assume there will always be 3 or more results in responseJson
+    // so we can limit our loop to both our upper limit 3 and the length of results
+    let { length = 0 } = responseJson;
+    if (length > 0) {
+        for (let i = 0; i < 3 && i < length; i++) {
+            $(targetedDiv).append(`<div class="beers"><h3>Beer Recomendations based of your recipe....</h3>
+            <h4>${responseJson[i].name}</h4>
+            <img src="${responseJson[i].image_url}" alt="" width="20%">
+            <p>${responseJson[i].description}</p></div>`)
+        }
+    } else {
+        $(targetedDiv).append(`<div><strong>no results</strong></div>`);
```

Comment:
otherwise, we show a useful message to the user that we couldn't get any useful results.